src/layouts/CoreLayout/CoreLayoutContainer.js

```diff
@@ -21,6 +21,6 @@ const mapDispatchToProps = { init }
 export default compose(
   connect(mapStateToProps, mapDispatchToProps),
   lifecycle({
-    componentDidMount() { this.props.init() },
+    componentDidMount() { /* this.props.init() */ },
```

Comment:
optional, better do not commit comments

Reply:
this file was removed
https://github.com/jibrelnetwork/jwallet-web/tree/feature/update-import-wallet/src/layouts/CoreLayout